### PR TITLE
CAT-2355 Removing Hard-Coding String

### PR DESCRIPTION
**Pull request by ndrnour**
In Pocket Code, program size is expressed in units of measurement (KB, MB, GB, etc.), these units are implemented as hard-coding string. Therefore, this pull request removes these hard_coding string from the app.

### CODE REVIEW

**Comment by robertpainsi** [comment]
@ndrnour, out of curiosity, how is `100 KB` translated into a RTL language in general?

**Comment by AyyalAwwad** [comment]
No parallel abbreviations for B, KB, MB, etc. in RTL languages, we should
use the complete words. for instance, KB is translated to Arabic (كيلو
بايت).


---------------------------------------------------------------------------------------------------------------------------------









*Aiman M. Ayyal Awwad .PhD. CandidateInstitute for Software
TechnologyFaculty of Computer Science and Biomedical EngineeringGraz
University of Technology8010 Graz, Inffeldgasse 13, 5th floor.AustriaTel :
+43 6604826205  Fax: +43 316 873-57 06E-mail: ayyalawwad@student.tugraz.at
<ayyalawwad@student.tugraz.at>, AwwadAiman@gmail.com <AwwadAiman@gmail.com>*

**Comment by robertpainsi** [comment]
Ah, good to know, thanks! Just one remaining question :smile: 
* Is the unit written on the left or right side of the number? `كيلوبايت 100` or `كيلوبايت I00`

**Comment by AyyalAwwad** [comment]
It is written on the left side of the number (كيلوبايت I00)

---------------------------------------------------------------------------------------------------------------------------------









*Aiman M. Ayyal Awwad .PhD. CandidateInstitute for Software
TechnologyFaculty of Computer Science and Biomedical EngineeringGraz
University of Technology8010 Graz, Inffeldgasse 13, 5th floor.AustriaTel :
+43 6604826205  Fax: +43 316 873-57 06E-mail: ayyalawwad@student.tugraz.at
<ayyalawwad@student.tugraz.at>, AwwadAiman@gmail.com <AwwadAiman@gmail.com>*

File: catroid/src/androidTest/java/org/catrobat/catroid/test/utils/RemovingHardCodingStringForObjectSizeUnit.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.clickInList(4);
+		//
+		solo.assertCurrentActivity("MainMenu",MainMenuActivity.class);
+		solo.sleep(2000);
+		solo.clickOnButton("البرامج");
```
Comment (cheidenreich) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/utils/RemovingHardCodingStringForObjectSizeUnit.java]: can you please put all those button names and expected strings into `static final string` variables with english names (e.g. BUTTON_NAME_PROGRAMS) at the top of this test class, so developers who cannot read these languages can understand what this test does

File: catroid/src/main/java/org/catrobat/catroid/utils/UtilFile.java [hunk 2]
```diff
@@ -78,25 +79,53 @@ public static long getProgressFromBytes(String projectName, Long progress) {
...
-
+/*
 	public static String getSizeAsString(File fileOrDirectory) {
 		final int unit = 1024;
 		long bytes = UtilFile.getSizeOfFileOrDirectoryInByte(fileOrDirectory);
 
 		if (bytes < unit) {
 			return bytes + " Byte";
-		}
+		}*/
```
Comment (cheidenreich) [inline catroid/src/main/java/org/catrobat/catroid/utils/UtilFile.java]: if this code will not be used anymore, please delete it

**Comment by robertpainsi** [comment]
> It is written on the left side of the number (كيلوبايت I00)

Thank you very much!

**Comment by thmq** [comment]
ok to test

**Comment by thmq** [comment]
this pr causes checkstyle warnings. to have a look at them please execute the gradle task check (found in the "verification" folder of your gradle console) or the task checkstyle (found in "other"). 

as a quick summary: the main problems relate to multiple blank lines in your code as well as incorrect whitespace wrapping around operators. furthermore: please do not change any files found in the ".idea" folder. (you can skip tracking for those files by executing the gradle task "gitSkipWorktreeForIntelijFiles - also found in the folder "other").

**Comment by thmq** [comment]
once you are done you can simply add your changes to this branch, the pr will be updated automatically. (consider using git commit --amend for this task 😉  - for more details have a look at [this](https://www.atlassian.com/git/tutorials/rewriting-history).

**Pull request closed without merging**